_posts/2018-04-10-personalized-search.md

```diff
@@ -0,0 +1,344 @@
+   rsp.addResponse(resultContext);
+}
+```
+
+## Response times
+We will also look at the response times:
+
+Response times for 95th percentile. Default search is search without any custom plugins, distribution search is the actual search we are trying to replace with personalization search. 
+Distribution search is a search where every xth (typically every 4th) document is a professional seller ad, 
+this is performed by doing one search for professional sellers and one for non-professional sellers, and merge them together by adding a professional ad for every xth position.
+The actual timings in the 95th percentile doubles from 225ms to 450ms. While the basic search takes around 80ms.
+
+<figure>
+    <img class="center-block" src="/images/2018-04-10-personalized-search/95th_percentile_latency.png" alt="alt" title="95th percentile latency" />
+    <figcaption style="text-align:right; font-style:italic;">95th percentile latency</figcaption>
+</figure>
+
+
+Lets also look at the 99th percentile for response time. We here removed the default search, because of too much noise, because of low traffic. 
+The personalization search's 99th percentile is between 60 to 100 ms slower than the search to be replaced.
+
+<figure>
+    <img class="center-block" src="/images/2018-04-10-personalized-search/99th_percentile_latency.png" alt="alt" title="99th percentile latency" />
+    <figcaption style="text-align:right; font-style:italic;">99th percentile latency</figcaption>
+</figure>
+
+## Search examples
+
+Here is a couple of examples of different user doing the same search at the same time, which will result in a completely different order of the ads:
+
+<figure>
+    <img class="center-block" src="/images/2018-04-10-personalized-search/example_with_personalized1.png" alt="alt" title="example 1 personalized search" />
+    <figcaption style="text-align:right; font-style:italic;">Personalized search with a user interested in antiques</figcaption>
+</figure>
+
+<figure>
+    <img class="center-block" src="/images/2018-04-10-personalized-search/example_with_personalized2.png" alt="alt" title="example 2 personalized search" />
+    <figcaption style="text-align:right; font-style:italic;">Personalized search with a user interested in furnitures for children</figcaption>
+</figure>
+
+We are now doing a personalization search for the 5 first pages of the search results. 
+Both of the searches includes the same 250 ads on the first 5 pages, but as you can see, the order is completely changed between the two examples.
+Published dates and the query relevance decides which subset of the search result to personalize.
+
+## Product results
+After testing the personalization search for around 3-4 months, we experienced around 2% better results for the some of the most important product KPIs for search:
+- Shorter time, from start of search to clicking a search result
+- More user sessions with clicking a search result
+- Overall more contacting the sellers
```

Comment:
Overall more buyers contacting the sellers ?
Overall more sellers were contacted ?
elns ? :)